### PR TITLE
Feature/631 search controls styles part 2

### PR DESCRIPTION
**Pull request by lraivio**
This PR:
- Adds MiniModal component for quickly adding small modals for example for filters.
- Adds search filter for people capacity. 

Refs #631.

### CODE REVIEW

File: app/pages/search/controls/PeopleCapacityControl.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+import ControlLabel from 'react-bootstrap/lib/ControlLabel';
+import FormGroup from 'react-bootstrap/lib/FormGroup';
+import FontAwesome from 'react-fontawesome';
```
Comment (YoYuUm) [inline app/pages/search/controls/PeopleCapacityControl.js]: - [ ] react-fontawesome package is missing in package.json

**Comment by lraivio** [comment]
Fixed and rebased with `feature/new-ui-and-helsinki-brand`.

**Comment by YoYuUm** [comment]
Are you sure you have add the changes? I don't see any modification to packaje.json

**Comment by YoYuUm** [comment]
Oh nevermind, I found the commit: https://github.com/City-of-Helsinki/varaamo/commit/6c14027489c7ad07dfed87867dbd63ff7b9c3ac1

File: app/pages/search/controls/PeopleCapacityControl.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  t: PropTypes.func.isRequired,
+  value: PropTypes.number,
+};
+
+function PeopleCapacityControl({ onChange, t, value }) {
+  return (
+    <div className="app-PeopleCapacityControl">
+      <MiniModal
+        buttonContent={
+          <span><FontAwesome name="users" /> {value || ''}</span>
```
Comment (YoYuUm) [inline app/pages/search/controls/PeopleCapacityControl.js]: - [ ] I don't see this icon in the browser

I've tried with other font-awesome icons and I don't see them either.

**Pull request merged**